### DISCUSSION TITLE
Presentation(s) at the CESM Workshop?

### DISCUSSION THREAD

**Discussion by nusbaume** [NCAR/ADF#162] [General]
Hi All,

@billsacks recently brought up the possibility of having one (or several) of us present on the ADF at the Software Engineering Working Group meeting as part of the CESM workshop this June.  It sounded like we could either have a single 3-5 minute update presentation, or possibly multiple presentations if we thought 3-5 minutes wasn't long enough.

What are people's thoughts on this?  I personally think that we could easily make a 5 minute presentation, but am not sure if we would want multiple presentations (I guess it depends on how much detail we want to go into)?  Of course if you have a different opinion on presenting, or think that we should present but have a specific request on what to present, then please let me know here.

Thanks!

**Comment by billsacks** [comment]
Having now heard back from other groups, I think my idea of a series of short presentations spanning a number of diagnostics packages doesn't necessarily make sense. I think it could make more sense to have a full-length (~ 10-15-minute) presentation focused just on updates on the ADF. This would be of interest to me and I think many others as well, so I would invite you to submit a talk on this to the SEWG session if you'd like.

**Comment by nusbaume** [comment]
Hi @billsacks , after our last ADF meeting everyone was happy with myself and @justin-richling giving a presentation on the ADF at the SEWG meeting.  However, I realized that we missed the presentation deadline (which was yesterday).  Could we still submit a presentation on the ADF, or is the SEWG meeting all booked up now?  If it is too late for us then no worries, I just figured it couldn't hurt to ask.  Thanks!

**Comment by billsacks** [comment]
Sounds great! The deadline was actually pushed back to next Monday so you haven't missed the boat yet.

**Reply by nusbaume** [reply]
Great!  I just submitted an abstract.  Thanks again for the presentation invite!
